### DISCUSSION TITLE
GitHub Suspended Students' Newly Created Accounts

### DISCUSSION THREAD

**Discussion by johannjhang** [community/community#121081] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

TL;DR: Do not create GitHub accounts at school or company together with the peers.

---
My wife recently joined a code camp and started learning programming.

She created her first GitHub account on 4/15. She also created two repositories in the same class: one empty, and the other stored some markdown files for practicing the markdown syntax. There were about 20 students in the classroom who likely followed the same process.

The next day, I guided her to use GitBook at home. She registered a GitBook account through her GitHub account's SSO, and created another repository for storing the content from GitBook. On the same day, I also guided her to enable the Visual Studio Code's settings sync through her GitHub SSO.

On 4/20, she discovered she could no longer log in to her GitBook account. We subsequently found out together that it was because her GitHub account had been suspended without any notice.

She has issued two support tickets to explain the situation and try to get back her account. She and her classmates must have created their accounts under the same IP address as they were in the same classroom. However, the support service kept her account suspended without explaining which policy she had violated, providing only a general statement that the account was created in an account farming effort. Moreover, as she is just a beginner, she might not even know how to proceed without my help as a software engineer.

I don't think using GitHub SSO on VS Code could be an issue. I am now publishing this page through GitBook and linking to a repository on my GitHub account, which serves as proof that this use case is safe. I believe the reason for this issue is that all the students in the class used the same IP address to create their individual accounts within a short period of time, as she later found out that other classmates encountered the same problem.

This is definitely a false positive decision. But I'm really surprised that she couldn't get back her account after issuing two tickets. They won’t even delete her account to allow her to reuse the username she is accustomed to.

[2024-04-23 GitHub Suspended Students' Newly Created Accounts | Johann Chang's Blog](https://bit.ly/3JAIPRU)
